setup.py

```diff
@@ -62,6 +62,7 @@
     'flake8',
     'flake8-docstrings==1.3.0',
     'flake8-import-order',
+    'gtimer',
```

Comment:
What is this used for?

Reply:
Its needed to run rlkit algorithm. Check `test_benchmark_td3.py`